src/shapes/Image.ts

```diff
@@ -856,5 +856,13 @@ export class Image<
   }
 }
 
-classRegistry.setClass(Image);
-classRegistry.setSVGClass(Image);
+classRegistry.setClass(FabricImage);
+classRegistry.setSVGClass(FabricImage);
+
+/**
+ * @deprecated The old fabric.Image class can't be imported as Image because of conflict with Web API.
+ * https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image
+ * For this reason it has been renamed to FabricImage.
+ * Please use `import { FabricImage }` in place of `import { Image as FabricImage }`
+ */
+export class Image extends FabricImage {}
```

Comment:
I think these 3 should reside in the main `fabric.ts` file and not in the def files